index.js

```diff
@@ -64,9 +64,10 @@ import EventEmitter from 'wolfy87-eventemitter'
       var anchor = actions[n];
       var port = anchor.getAttribute("data-port");
       var protocol = anchor.getAttribute("data-protocol") || 'http:';
+      var hostPrefix = anchor.getAttribute("data-host-prefix") + '-' || '';
```

Comment:
This won't work since it'll add the `-` always even if `anchor.getAttribute` is not defined

Reply:
omg you're right ! I need to stop coding while eating :)